app/views/citizens/accounts/index.html.erb

```diff
@@ -41,7 +41,7 @@
 
     <% end %>
 
-    <p><%= link_to_accessible t(".link_other_account"), citizens_banks_path, class: "govuk-link" %></p>
+    <p><%= govuk_link_to t(".link_other_account"), citizens_banks_path, class: "govuk-link" %></p>
```

Comment:
I think the class `govuk-link` can be removed as it is applied by default by the helper `govuk_link_to`

Reply:
Thank you for the tip. I have made these suggested changes now.